src/sentry/api/endpoints/project_stacktrace_link.py

```diff
@@ -258,6 +292,31 @@ def get(self, request: Request, project: Project) -> Response:
                     # When no code mapping have been matched we have not attempted a URL
                     if current_config["outcome"].get("attemptedUrl"):
                         result["attemptedUrl"] = current_config["outcome"]["attemptedUrl"]
+
+                try:
```

Comment:
Move this block into a function and call it here and put it inside of an else condition with the if condition right above.